clap-utils/src/lib.rs

```diff
@@ -23,6 +23,10 @@ impl std::fmt::Debug for DisplayError {
     }
 }
 
+pub fn hidden_unless_forced() -> bool {
+    std::env::var("SOLANA_NO_HIDDEN_CLI").is_err()
```

Comment:
May I humbly suggest:
```suggestion
    std::env::var("SOLANA_NO_HIDDEN_CLI_ARGS").is_err()
```